src/ReactiveDomain/ReactiveDomain.Tests/EventStore/GetEventStoreRepositoryIntegrationTests.cs

```diff
@@ -18,7 +18,7 @@ namespace ReactiveDomain.Tests.EventStore
 
     public class GetEventStoreRepositoryIntegrationTests : IDisposable
     {
-       
+        private const string SkipReason = "EventStore";
         /// <summary>
```

Comment:
The way these test are written they only run if there is a live version of the event store running. this cannot be assumed by the auto runner we are using. If it can get converted to use the inMemory runner then that can be changed.

Reply:
that wasn't my question. I understand why the tests are skipped. I don't understand why you made SkipReason a const that looks like maybe it's planned to be a variable in the future.